### DISCUSSION TITLE
Input And TextArea can support copy ？

### DISCUSSION THREAD

**Discussion by jinphic** [ant-design/ant-design#42633] [Q&A]
copy content is common in business

**Comment by bombillazo** [comment]
This would be great

**Comment by BoyYangzai** [comment]
@jinphic  hi bro
That's a easy feature but maybe we recommend that you manually wrap a simple `CopyButton` component to implement the function of copying input or textArea😄
https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript

**Reply by jinphic** [reply]
in some case , like formily with schema, if i want a input with copy, I need to write a new InputCopy to cover the formily/antd/Input.
It requires a significant amount of code and is too common in the business. 
In my opinion, this can be used as a built-in capability.
And I have pr code to 'rc-input', hope to pass.

**Reply by jinphic** [reply]
It's not a business requirement, but a basic feature I think needed by many antd users

**Reply by BoyYangzai** [reply]
@zombieJ @afc163 @MadCcc cc

**Reply by jinphic** [reply]
cc

**Reply by FrenchMajesty** [reply]
I support this notion. It's very common to want to copy/paste text in a large textarea or code snippet. In fact, antd's own documentation uses it repeatedly. It would nice to have that built in.
